CODEOWNERS

```diff
@@ -76,51 +74,49 @@ CODEOWNERS   @Intel-Media-SDK/mediasdk-admin
 /_studio/shared/umc/codec/vp9_dec/ @Intel-Media-SDK/mediasdk-admin will.guo@intel.com yan.wang@intel.com
 
 # vp9e:
-/_studio/vp9e_hw/ @Intel-Media-SDK/mediasdk-admin ekaterina.shiryaeva.alexandrovna@intel.com anastasiia.sokolova@intel.com
-_studio/mfx_lib/encode_hw/vp9/ @Intel-Media-SDK/mediasdk-admin ekaterina.shiryaeva.alexandrovna@intel.com anastasiia.sokolova@intel.com
+/_studio/vp9e_hw/ @Intel-Media-SDK/mediasdk-admin chunbo.chen@intel.com apple.wang@intel.com
+_studio/mfx_lib/encode_hw/vp9/ @Intel-Media-SDK/mediasdk-admin chunbo.chen@intel.com apple.wang@intel.com
 
 # vpp:
 /_studio/mfx_lib/vpp/ @Intel-Media-SDK/mediasdk-admin furong.zhang@intel.com
 /_studio/shared/umc/codec/color_space_converter/ @Intel-Media-SDK/mediasdk-admin furong.zhang@intel.com
 /_studio/shared/umc/codec/scene_analyzer/ @Intel-Media-SDK/mediasdk-admin furong.zhang@intel.com
 
 # tests:
-/tests/ @Intel-Media-SDK/mediasdk-admin dmitry.v.rogozhkin@intel.com dmitry.gurulev@intel.com
+/tests/ @Intel-Media-SDK/mediasdk-admin dmitry.v.rogozhkin@intel.com apple.wang@intel.com
 
 # avc_fei:
-/_studio/mfx_lib/fei/ @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
-/samples/sample_fei/ @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
+/_studio/mfx_lib/fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/samples/sample_fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
 
 # hevc_fei:
-/_studio/hevc_fei/ @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
-/samples/sample_hevc_fei/ @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
-/_studio/mfx_lib/encode_hw/h265/src/*fei* @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
-/_studio/mfx_lib/encode_hw/h265/include/*fei* @Intel-Media-SDK/mediasdk-admin miroslav.goncharenko@intel.com dmitry.ermilov@intel.com leonid.a.kulakov@intel.com
+/_studio/hevc_fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/samples/sample_hevc_fei/ @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/_studio/mfx_lib/encode_hw/h265/src/*fei* @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
+/_studio/mfx_lib/encode_hw/h265/include/*fei* @Intel-Media-SDK/mediasdk-admin thomas.li@intel.com apple.wang@intel.com
 
 # enctools:
-/_studio/enctools/ @Intel-Media-SDK/mediasdk-admin timofei.a.kulakov@intel.com olga.golikova@intel.com
-/_studio/enctools/aenc @Intel-Media-SDK/mediasdk-admin sergey.plotnikov@intel.com
-/_studio/shared/include/mfxenctools-int.h @Intel-Media-SDK/mediasdk-admin timofei.a.kulakov@intel.com olga.golikova@intel.com
+/_studio/enctools/ @Intel-Media-SDK/mediasdk-admin dmitry.v.rogozhkin@intel.com
```

Comment:
We need appropriate owner for enctools part. I am not the one. sergey.plotnikov@intel.com might be the candidate.